pkg/internal/config/config.go

```diff
@@ -106,8 +106,8 @@ func New(baseConfig kconfig.Config) (Config, error) {
 	if rawNUMASocket = baseConfig.Params[NUMASocketParamName]; rawNUMASocket == "" {
 		return Config{}, ErrInvalidNUMASocket
 	}
-	numaSocket, err := strconv.Atoi(rawNUMASocket)
-	if err != nil || numaSocket < 0 {
+	numaSocket, err := parseNonNegativeInt(rawNUMASocket)
+	if err != nil {
```

Comment:
This change is not documented in the commit's message.
Please consider creating a new commit for it.

Reply:
changed the current commit to document it